### PR TITLE
add warning when leaving form settings

### PR DESCRIPTION
**Pull request by Crabcyborg**
Fixes https://github.com/Strategy11/formidable-pro/issues/2764

### CODE REVIEW

File: js/formidable_admin.js [hunk 1]
```diff
@@ -6786,6 +6791,9 @@ function frmAdminBuildJS() {
 
 			jQuery( document ).on( 'frm-multiselect-changed', '#protect_files_role', adjustVisibilityValuesForEveryoneValues );
 
+			jQuery( document ).on( 'submit', '.frm_form_settings', settingsSubmitted );
+			jQuery( document ).on( 'change', '#form_settings_page input:not(.frm-search-input), #form_settings_page select', fieldUpdated );
```
Comment (jairoprez) [inline js/formidable_admin.js:6795]: Warning is working properly with input type fields but I think you forgot to cover the textarea fields https://recordit.co/nqOgv4f7IO, this should do the trick `#form_settings_page textarea`

**Review by jairoprez — COMMENTED** [review]
@Crabcyborg This looks good to me, I found just a small detail that was overlooked and can be easily resolved I think

File: js/formidable_admin.js (hunk 1 above)
Comment (Crabcyborg) [reply js/formidable_admin.js:6795]: Ohhh nice catch @jairoprez !

**Comment by Crabcyborg** [comment]
@jairoprez thanks for considering the textarea. I neglected them in the original update, and added to that as well. Took a second to hunt one down, but it looks like you wouldn't get the warning after editing content for an HTML field either.

**Comment by jairoprez** [comment]
@Crabcyborg Nice job! - select fields are now covered.

**Comment by jairoprez** [comment]
Hey @stephywells this PR looks good to me.

**Comment by stephywells** [comment]
Thanks you two! This looks great!

**Pull request merged**